--- ../ios/Pods/boost/boost/container_hash/hash.hpp.orig.hpp	2024-07-10 14:24:31
+++ ../ios/Pods/boost/boost/container_hash/hash.hpp	2024-07-10 14:25:23
@@ -128,7 +128,7 @@
         };
 #else
         template <typename T>
-        struct hash_base : std::unary_function<T, std::size_t> {};
+    struct hash_base : std::__unary_function<T, std::size_t> {};
 #endif
 
         struct enable_hash_value { typedef std::size_t type; };
