--- ../node_modules/react-native/ReactCommon/yoga/yoga/Yoga.cpp.orig	2024-07-10 13:35:29
+++ ../node_modules/react-native/ReactCommon/yoga/yoga/Yoga.cpp	2024-07-10 11:54:48
@@ -2229,7 +2229,7 @@
         depth,
         generationCount);
     node->setLayoutHadOverflow(
-        node->getLayout().hadOverflow() |
+        node->getLayout().hadOverflow() ||
         currentRelativeChild->getLayout().hadOverflow());
   }
   return deltaFreeSpace;
